src/modules/layout/components/LocationFooter.js

```diff
@@ -24,9 +24,10 @@ export class LocationFooter extends React.PureComponent<PropsType> {
       <CleanLink to={new DisclaimerRouteConfig().getRoutePath({ city, language })}>
         {t('imprintAndContact')}
       </CleanLink>
+      <CleanAnchor href='https://integreat-app.de/about/'>{t('settings:about')}</CleanAnchor>
```

Comment:
Look [here](https://react.i18next.com/latest/withtranslation-hoc#withtranslation-params) if you have questions to this syntax